### PR TITLE
Draft fp 0509

### PR DESCRIPTION
**Pull request by JuliaWindmiller1987**
Very first draft of transfer flight with an EarthCARE overpass close to Sal and three circles along the spine of the ITCZ. One circle is designated for coordination with the RV METEOR.

### CODE REVIEW

File: orcestra_book/plans/HALO-20240905a.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+landing: 2024-09-05 tba
+orphan: true
+platform: HALO
+takeoff: 2024-09-05 tba
```
Comment (lkluft) [inline orcestra_book/plans/HALO-20240905a.md:34]: While I appreciate the honesty in the accuracy of the takeoff and landing times, our scripts need them to be valid ISO 8601 strings 😬.

**Review by lkluft — CHANGES_REQUESTED** [review]

File: orcestra_book/plans/HALO-20240905a.md (hunk 1 above)
Comment (karstenpeters) [reply orcestra_book/plans/HALO-20240905a.md:34]: I also already saw that ;-) but was too slow to reply

File: orcestra_book/plans/HALO-20240905a.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
+---
+arrival_airport: GVAC
```
Comment (karstenpeters) [inline orcestra_book/plans/HALO-20240905a.md:2]: Should this not be Barbados?

**Comment by JuliaWindmiller1987** [comment]
Made the requested changes :) Thanks!

**Review by lkluft — APPROVED** [review]

**Review by bjorn-stevens — COMMENTED** [review]
This seems fine and builds for me. 

On the presentation; I am not sure how you want to handle this.  Tobi and I have been fine tuning the flight planning to try and make it simpler and easier to read.  My 09-03 flight plan takes a step in this direction, but it still incorporates some in-exactness, that is awaiting what he calls a "snap-to" satellite track, which will fix some of the minor inconsistencies that you might note.  

He is also adding a 'to-txt' functionality.  This will get everything in place for the rookie flight planners.

**Pull request merged**